src/transformer/decode.ts

```diff
@@ -348,19 +348,25 @@ export class Decoder {
         throw new MissingElement(model.containedObjectID)
       }
 
-      const figcaption: FigCaptionNode = this.getFigcaption(model)
-
       return schema.nodes.equation_element.createChecked(
         {
           id: model._id,
-          suppressCaption: Boolean(model.suppressCaption),
+          title: model.title,
           suppressTitle: Boolean(
             model.suppressTitle === undefined ? true : model.suppressTitle
           ),
         },
-        [equation, figcaption]
+        [equation]
       ) as EquationElementNode
     },
+    [ObjectTypes.InlineMathFragment]: (data) => {
+      const model = data as InlineMathFragment
+
+      return schema.nodes.inline_equation.createChecked({
+        id: model._id,
+        content: model.content,
+      }) as InlineEquationNode
+    },
```

Comment:
This should not be needed; Instead, we should skip encoding inline math in its own model. In encode, I added a check to skip all children of paragraph except inline math, so we should update it. Maybe we can try replacing that check with another that skips all inline nodes, since inline by definition means it should not be a standalone model.